brayns/Brayns.cpp

```diff
@@ -113,6 +108,54 @@ struct Brayns::Impl
             _finishLoadScene();
     }
 
+    void addPlugins()
+    {
+#ifdef BRAYNS_USE_NETWORKING
+        auto plugin{std::make_shared<RocketsPlugin>(_engine, this)};
+        _extensionPluginFactory.add(plugin);
+        _actionInterface = plugin;
+#endif
+#ifdef BRAYNS_USE_DEFLECT
+        _extensionPluginFactory.add(
+            std::make_shared<DeflectPlugin>(_engine, this));
+#endif
+    }
+
+    void loadPlugins()
+    {
+#ifdef BRAYNS_USE_OSPRAY
```

Comment:
Is it a problem we need to have ospray enabled to load our plugins?

Reply:
Today nothing works without having ospray anyways. But for your plugins you don't need/see it. We can also use lunchbox::DSO for the library loading. Or some boost stuff.